docs/migrate-to-vector-tiles-v1.md

```diff
@@ -0,0 +1,246 @@
+# Migrate to vector tiles 1.0 from earlier releases
+
+If you are connecting to the Mapzen vector tile service using a URL that starts with `https://vector.mapzen.com`, you need to consider your migration path for Mapzen vector tiles 1.0 and take action.
+
+The 1.0 release of the Mapzen vector tiles service has some [major changes and enhancements](https://mapzen.com/blog/v1-vector-tile-service/), and Mapzen recommends all existing users of the service migrate to it. This process is not automatic, and does require you to modify your project.
+
+At a minimum, you need to update your code or stylesheets, such as [Tangram YAML files](https://mapzen.com/documentation/tangram/sources/), to point to the service's new URL. Any additional work to use the 1.0 release depends on how much customization and styling you are using.
+
+## Update the service URL
+
+You can determine which release of the service you are using by looking at the URL in your code. If you see `vector.mapzen.com`, you are using the deprecated tile service.
+
+To use vector tiles 1.0, replace your URL with the following scheme, depending on which tile format you want:
+
+- GeoJSON: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.json`
+- TopoJSON: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson`
+- Mapbox Vector Tile: `http://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.mvt`
+
+You need to append your own [API key](https://mapzen.com/documentation/overview/) at the end for access to the full rate limits.
+
+Mapzen offers several different types of tiles in vector and raster formats and the service combines data from multiple sources. Because of this, it was not possible to redirect existing queries automatically to use the 1.0 URL.
```

Comment:
Primarily it's not possible to redirect because of backwards incompatible data model changes between v0.10 and v1.0.